ui/pages/v2/HelmReleasePage.tsx

```diff
@@ -11,6 +13,31 @@ type Props = {
   className?: string;
 };
 
+function helmChartLink(helmRelease: HelmRelease) {
```

Comment:
Does this need to be exported and then imported into HelmReleaseDetail now? Or deleted here? I'm just seeing the warning (defined but never used) and want to make sure this still works

Reply:
Nice catch. I had turned off my `eslint` to save cycles. Not sure how that passed CI